src/main/java/cc/hyperium/mods/browser/BrowserMod.java

```diff
@@ -79,37 +96,44 @@ public AbstractMod init() {
         }
         browserGui = new GuiBrowser(homePage);
 
-//        addShortcutKeys();
         registerCommands();
         Multithreading.runAsync(() -> {
             long start = System.currentTimeMillis();
-            JFrame jFrame = new JFrame("Keycode Initializer");
+            JFrame jFrame = new JFrame("Ignore Me! I'm for Hyperium.");
```

Comment:
Not sure about this, users should perhaps know to some extent what this window does.